test/ng/httpSpec.js

```diff
@@ -2364,17 +2364,45 @@ describe('$http param serializers', function() {
       expect(decodeURIComponent(jqrSer({a: 'b', foo: ['bar', 'baz']}))).toEqual('a=b&foo[]=bar&foo[]=baz');
     });
 
+    it('should serialize arrays with functions', function() {
+      expect(jqrSer({foo:[function() { return 'bar'; }]})).toEqual('foo%5B%5D=bar');
```

Comment:
Missing space after `:`. :grin:
BTW, you an probably use `valueFn('bar')` to save some typing 😎